tests/contrib/hooks/test_gcp_dataflow_hook.py

```diff
@@ -111,6 +138,55 @@ def test_start_python_dataflow(self, mock_conn,
         self.assertListEqual(sorted(mock_dataflow.call_args[0][0]),
                              sorted(expected_cmd))
 
+    @mock.patch(DATAFLOW_STRING.format('uuid.uuid4'))
```

Comment:
This is a lot of code duplication. We should change it to "@parameterized" tests.

Reply:
Sure thanks for the tip.